### ISSUE TITLE
Support Link Abbreviations

### ISSUE THREAD

**Issue by amake** [amake/orgro#105]
https://orgmode.org/manual/Link-Abbreviations.html

In other words interpret `#+LINK:` directives so that you can do e.g.

```org
#+LINK: duckduckgo https://duckduckgo.com/?q=%s

[[duckduckgo:OrgMode]]
```

**Comment by amake** [comment]
This is implemented in v1.42.0, available for testing soon:

- [iOS](https://testflight.apple.com/join/8YTxcuZC)
- [Android](https://play.google.com/store/apps/details?id=com.madlonkay.orgro)

**Comment by amake** [comment]
This is now available on all platforms.
